source/MRMesh/MRIOParsing.cpp

```diff
@@ -172,7 +173,7 @@ VoidOrErrStr parseObjCoordinate( const std::string_view& str, Vector3<T>& v, Vec
         );
     }
     if ( !r )
-        return unexpected( "Failed to parse vertex" );
+        return unexpected( "Failed to parse vertex: " + std::string( removeSuffixWithNewline( str, 80 ) ) );
```

Comment:
Better to introduce a separate "trimRight" function instead of a complex "trimRight + substr" one. Boost already has a similar `boost::algorithm::trim_right_copy` function.